### DISCUSSION TITLE
Gutenberg W3C validation

### DISCUSSION THREAD

**Discussion by solidcodes** [WordPress/gutenberg#43212] [Developer Experience]
Hi guys,

This is my first time using Gutenberg.
Just 100% Gutenberg no templates.
I use twentytwentytwo theme.

The name of the site I created using Gutenberg is https://webmasterseoph.com/
and I validated it at W3C https://validator.w3.org/

I tried validating the site I created using Gutenberg and I got these weird errors.
`Error: CSS: --wp--custom--spacing--medium: Invalid type: 4 * var(--wp--style--block-gap).
At line 81, column 5201
le--block-gap)));--wp--custom-`

and this is another error,
`Error: Element [style](https://html.spec.whatwg.org/multipage/#the-style-element) not allowed as child of element [body](https://html.spec.whatwg.org/multipage/#the-body-element) in this context. (Suppressing further errors from this subtree.)

From line 352, column 3; to line 352, column 31

</div>↩↩		<style id="skip-link-styles">↩		.sk

Contexts in which element [style](https://html.spec.whatwg.org/multipage/#the-style-element) may be used:
Where [metadata content](https://html.spec.whatwg.org/multipage/#metadata-content-2) is expected.
In a [noscript](https://html.spec.whatwg.org/multipage/#the-noscript-element) element that is a child of a [head](https://html.spec.whatwg.org/multipage/#the-head-element) element.
Content model for element [body](https://html.spec.whatwg.org/multipage/#the-body-element):
[Flow content](https://html.spec.whatwg.org/multipage/#flow-content-2).`

I hope the team of Gutenberg programmers will make Gutenberg ERROR FREE in W3C validaiton.

Thank you very much in advance.

**Comment by skorasaurus** [comment]
Regarding the 2nd error, "`Error: Element [style](https://html.spec.whatwg.org/multipage/#the-style-element) not allowed as child of element [body](https://html.spec.whatwg.org/multipage/#the-body-element) in this context. (Suppressing further errors from this subtree.)" is commonly encountered that is being tracked at https://github.com/WordPress/gutenberg/issues/38917 

As for the first error, I ran your website through the validator and that error did not come up.

**Reply by solidcodes** [reply]
These problems already solved.
The problem was solved using a plugin called LiteSpeed.
hope this helps to others too.
